src/thunderbots/software/ai/passing/pass.h

```diff
@@ -0,0 +1,101 @@
+/**
+ * Declaration for the "Pass" Class
+ */
+
+#include <array>
+#include <cstdlib>
+#include <iostream>
+
+#include "geom/point.h"
+#include "util/timestamp.h"
+
+namespace AI::Passing
+{
+    // TODO: simple test for getters
+    /**
+     * This class represents a Pass, with a given start position, end position,
+     * speed, and start time
+     */
+    class Pass
+    {
+       public:
+        /**
+         * Creates a pass with all 0 values
+         */
+        // Pass();
+
+        /**
+         * Create a pass with given parameters
+         *
+         * @param passer_point The point the pass should start at
+         * @param receiver_point The point the receiver should be at to receive the pass
+         * @param pass_speed_m_per_s The speed of the pass, in meters/second
+         * @param pass_start_time The time that the pass should start at
+         */
+        Pass(Point passer_point, Point receiver_point, double pass_speed_m_per_s,
+             Timestamp pass_start_time);
+
+        /**
+         * Gets the value of the receiver point
+         *
+         * @return The value of the receiver point
+         */
+        Point receiverPoint();
+
+        /**
+         * Gets the value of the passer point
+         *
+         * @return The value of the passer point
+         */
+        Point passerPoint();
+
+        /**
+         * Gets the value of the pass speed
+         *
+         * @return The value of the pass speed, in meters/second
+         */
+        double speed();
+
+        /**
+         * Gets the value of the pass start time
+         *
+         * @return The value of the pass start time
+         */
+        Timestamp startTime();
+
+        /**
+         * Implement the "<<" operator for printing
+         *
+         * This is implemented in the header because there were namespace issues with
+         * putting it in the `.cpp`.
+         *
+         * @param output_stream The stream to print to
+         * @param pass The pass to print
+         * @return The output stream with the string representation of the class appended
+         */
+        friend std::ostream& operator<<(std::ostream& output_stream, const Pass& pass);
+        //{
```

Comment:
Seems like this is now implemented in the `.cpp` file. Can this commented-out code be removed? The javadoc comment should be updated if so.

Reply:
yeeep, wow, I really should've gone over this more closely before requesting review